src/resizer/include/resizer/SteinerTree.hh

```diff
@@ -118,6 +119,8 @@ public:
   SteinerPt left(SteinerPt pt);
   SteinerPt right(SteinerPt pt);
   void findLeftRights(const Network *network);
+  void dumpSVG(const Network *network,
```

Comment:
if (debug_->check("write_steiner_svg", 1))

and then sta::set_debug "write_steiner_svg" 1 in the command file